src/function/scalar/list/list_contains.cpp

```diff
@@ -0,0 +1,226 @@
+#include <iostream>
+#include "duckdb/planner/expression/bound_function_expression.hpp"
+#include "duckdb/function/scalar/nested_functions.hpp"
+#include "duckdb/planner/expression_binder.hpp"
+
+namespace duckdb {
+
+static void SetResultFalse(Vector &result) {
+	auto result_data = FlatVector::GetData<bool>(result);
+	result_data[0] = false;
+	return;
+}
+
+template <class T>
```

Comment:
This function does not have to be a templated function if I understand correctly. This function only deals with `string_t`.